app/src/main/java/com/example/music_app/ui/models/utils/RetrofitAuthService.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.music_app.ui.models.utils
```

Comment:
Please move network classes to the separate package, `utils` is not acceptable for Retrofit interfaces